.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/hfjn/.local/share/virtualenvs/esque-YsmoIp6f/bin/python"
```

Comment:
This is probably not supposed to end up here either ;)

Reply:
God damnit. How did that came back?